docs/core/tools/dotnet-tool-restore.md

```diff
@@ -9,7 +9,7 @@ ms.date: 02/14/2020
 
 ## Name
 
-`dotnet tool restore` - Installs on your machine the .NET local tools that are in scope for the current directory.
+`dotnet tool restore` - Installs the .NET local tools onto your machine that are in scope for the current directory.
```

Comment:
Does getting rid of machine completely make it even clearer? What do you think?
```suggestion
`dotnet tool restore` - Installs the .NET local tools that are in scope for the current directory.
```

Reply:
Makes more sense, yeah